src/components/Tabs/Tab.tsx

```diff
@@ -50,9 +24,10 @@ const Tab: React.FC<TabProps> = ({
 
   return (
     <Label
+      $color={color}
       $isSelected={isSelected}
       as={isLink ? RouterLink : 'a'}
-      color={color}
+      size={size}
```

Comment:
any reason you made color transient but not size?

Reply:
@vaclav-zeman size does not spill into DOM, but color gets passed into styled